react-events/my-app/src/CustomButton.js

```diff
@@ -0,0 +1,6 @@
+export default function CustomButton({text, color, onCustomClick}) {
+
+  return ( <button  onClick= {() => {onCustomClick(text)}}
```

Comment:
The `{}` around `onCustomClick(text)` are OK but not necessary. The `{}` there indicate the arrow function's body. If they are not there, it is an expression, which is legal syntax.

This is a tiny issue. I'm just letting you know what is more common in React code.